ui/jobs/jobs.js

```diff
@@ -2396,6 +2408,14 @@ class Bars {
             if (m)
               this.buffTracker.onUseAbility(m.groups.id, log);
           }
+          // use of GP Potion
+          let cordialRegex = new RegExp(this.me + ':20(017FD|F5A3D|F844F|0420F|0317D):');
```

Comment:
Can you make this Regexes.ability({source: this.me, id: '20(017FD|F5A3D|F844F|0420F|0317D)'})?

I agree that this will need to be updated in the future, but this also doesn't need to be translated.  Arguably, the game text for gaining GP might also change in the future (although probably less likely than adding new cordials)?

Reply:
I don't think game text will change. However, if you want this way, I'll do that way.